CHANGELOG.md

```diff
@@ -7,6 +7,7 @@ CHANGELOG
 **ENHANCEMENTS**
 - Add new configuration parameter `DeletionPolicy` for EFs and FSx for Lustre shared storage 
   to support storage retention on deletion.
+- Encrypt the `EcrImageBuilderSNSTopic` used to communicate docker image build events using Server-Side Encryption.
```

Comment:
nit: be explicit that this change is in the API infrastructure template. e.g. Enable server side encryption for the SNS topic created when deploying ParallelCluster API and used to communicate docker image build events.